README.md

```diff
@@ -754,6 +754,25 @@ endfunction
 let g:test#custom_alternate_file = function('CustomAlternateFile')
 ```
 
+## Overriding test commands
+
+This is considered an advanced feature, subject to active development and further changes.
```

Comment:
It needs to be made clear that this feature overrides the zero configuration approach

```suggestion
This is considered an advanced feature, subject to active development and further changes. It overrides the zero configuration approach, and requires you to manually configure the test runners.
```

Reply:
Sure, done and same in doc